docs/helm/helm_repo_add.md

```diff
@@ -8,7 +8,7 @@ add a chart repository
 add a chart repository
 
 ```
-helm repo add [flags] [NAME] [URL]
+helm repo add [flags] [NAME] [URL] [USERNAME] [PASSWORD]
```

Comment:
Would you mind making these flags instead of args? The general rule of thumb we use are `args should be required and flags are optional`. This will also make the experience consistent across the other commands that can take a username and password.

Reply:
I changed them to be flags.